### PR TITLE
New: Docs - Version update logic

### PR DESCRIPTION
**Pull request by Nevoss**
ref: ED-2302

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+### Elementor Pro
+
+For the Elementor pro updates, the class "Pro_Version_Control" interrupt the HTTP request that sends to Elementor servers (to check if there is a new version to update) and adds the "dev" param to inform the Elementor servers that the current installation is subscribed to the ***dev*** channel.
+
+### Conclution
+
+The main difference between the 2 approaches is that in the core version of Elementor the logic of which is the right version for the update is calculated in the "dev" plugin. in the pro version of Elementor, the remote server has the responsibility to deliver the right version.
+
+Elementor Beta (Developer Edition) has responsibility for some UI changes that make the plugin look bit different from the stable version of Elementor.
```
Comment (StyleShit) [inline docs/README.md:19]:
```suggestion
Elementor Beta (Developer Edition) also adds some UI changes which makes the plugin look a bit different from the stable version of Elementor.
```

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
-### Conclution
+### Conclusion
 
-The main difference between the 2 approaches is that in the core version of Elementor the logic of which is the right version for the update is calculated in the "dev" plugin. in the pro version of Elementor, the remote server has the responsibility to deliver the right version.
+The main difference between the 2 approaches is that in the Core version of Elementor the logic of which is the right version for the update determined in the plugin itself, while in the Pro version of Elementor, the remote server has the responsibility to deliver the right version.
 
-Elementor Beta (Developer Edition) has responsibility for some UI changes that make the plugin look bit different from the stable version of Elementor.
+Elementor Beta (Developer Edition) also adds for some UI changes which makes the plugin look a bit different from the stable version of Elementor.
```
Comment (StyleShit) [inline docs/README.md:19]:
```suggestion
Elementor Beta (Developer Edition) also adds some UI changes which makes the plugin look a bit different from the stable version of Elementor.
```

Comment (StyleShit) [reply docs/README.md:19]: My bad..

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
Comment (Nevoss) [reply docs/README.md]: I should have noticed it.

**Review by StyleShit — APPROVED** [review]

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Elementor Beta (Developer Edition) Docs 
+
+The module "developer-edition" creates 2 instances of the classes `Core_Version_Control` and `Pro_Version_Control`, the main idea behind those classes is to interrupt the WordPress update mechanism and make sure that the future updates of "Elementor" and "Elementor Pro" comes from the "dev" channel.
```
Comment (matipojo) [inline docs/README.md:3]:
```suggestion
The module "developer-edition" creates instances of the classes `Core_Version_Control` and `Pro_Version_Control`, the main idea behind those classes is to interrupt the WordPress update mechanism and make sure that the future updates of "Elementor" and "Elementor Pro" comes from the "dev" channel.
```

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+The class `Core_Version_Control`, which responsible for Elementor updates, removes all the additional filters of `pre_set_site_transient_update_plugins` that controls the updates and registers a new filter that sends an API request to the wordpress.org plugin info endpoint, it gets all Elementor versions and checks which one of them is the last ***dev*** version of Elementor.
+
+### Elementor Pro
+
+For the Elementor pro updates, the class `Pro_Version_Control` interrupts the HTTP request that was being send to Elementor servers (to check if there is a new version to update) and adds the `dev` param to inform the Elementor server that the current installation subscribed to the ***dev*** channel.
+
+### Conclusion
+
+The main difference between the 2 approaches is that in the Core version of Elementor the logic of which is the right version for the update determined in the plugin itself, while in the Pro version of Elementor, the remote server has the responsibility to deliver the right version.
```
Comment (matipojo) [inline docs/README.md:17]:
```suggestion
The main difference between the 2 approaches is that in the Core version of Elementor the decision of which is the right version for the update is determined in the plugin itself, while in the Pro version of Elementor, the remote server has the responsibility to deliver the right version.
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Pull request merged**